src/components/layout.jsx

```diff
@@ -123,7 +129,7 @@ class Layout extends React.Component {
           <div className='col-md-4'>
             <div className='header'>
               <h1 className='title'>
-                <Link to='/'>FreeFeed-beta</Link>
+                <IndexLink to='/' onClick={this.handleLogoClick.bind(this)}>FreeFeed-beta</IndexLink>
```

Comment:
this can be done without explicit binding. just use arrow-function instead of method (`handleLogoClick = () => { …`)